webapp/src/resources.js

```diff
@@ -71,6 +72,7 @@ export const blockTypes = {
   ms: { description: "Mass Spectrometry", component: MassSpecBlock, name: "Mass Spectrometry" },
   chat: { description: "Virtual assistant", component: ChatBlock, name: "Virtual Assistant" },
   ftir: { description: "FTIR", component: BokehBlock, name: "FTIR" },
+  uvvis: { description: "UV-Vis", component: UVVisBlock, name: "UV-Vis" },
```

Comment:
```suggestion
  "uv-vis": { description: "UV-Vis", component: UVVisBlock, name: "UV-Vis" },
```

Can we use the hyphenated version everywhere please? (both in the Python and JS)